transaction_adaptor.py

```diff
@@ -0,0 +1,93 @@
+
+import csv
+import string
+import datetime
+
+def discover_adaptor(file_path : str):
+    '''
+    Discover
+    export as CSV file
+    Trans. Date, Post Date, Description, Amount, Category
+    '''
+    transaction_list = []
+    with open(file_path) as discover_csv:
+        discover_data = list(csv.DictReader(discover_csv))
+    
+    for row in discover_data:
+        transaction_list.append(make_transaction_from_discover(row))
+    return transaction_list
+
+def make_transaction_from_discover(discover_transaction):
+    date = change_date(discover_transaction["Trans. Date"])
```

Comment:
Different institutions might format their dates differently, so it'd be very easy for `change_date` to accidentally do the wrong thing.

This would be better: `datetime.datetime.strptime(discover_transaction["Trans. Date"], "%m/%d/%Y").date()`